tbx/project_styleguide/templates/patterns/pages/work/work_page.html

```diff
@@ -10,14 +10,26 @@
 {% endblock %}
 
 {% block main_image %}
+    {% if page.remove_desaturation_on_header_image %}
+        {% image page.header_image fill-1600x700 format-webp as desktop_image %}
+        {% image page.header_image fill-1280x560 format-webp as desktop_image_retina %}
```

Comment:
I'm not quite confident in how I've converted the srcset sizes.

Reply:
It also looks like the maximum the desktop image can be is 1680px, so I'm confused why the `srcset` had 1600px set up for desktop sizes.

Reply:
Now updated